### PR TITLE
Add icon and graphical-icon components

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: app/javascript/components/common/GraphicalIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export function GraphicalIcon({ icon }: { icon: string }) {
+  return (
+    <svg className="c-graphical-icon" role="img">
```
Comment (iHiD) [inline app/javascript/components/common/GraphicalIcon.tsx]:
```suggestion
    <svg className="c-graphical-icon" role="presentation">
```

File: app/javascript/components/common/index.ts [hunk 2]
```diff
@@ -1 +1,5 @@
 export { CopyToClipboardButton } from './CopyToClipboardButton'
+export { GraphicalIcon } from './GraphicalIcon'
+export { Icon } from './Icon'
+export { Loading } from './Loading'
```
Comment (iHiD) [inline app/javascript/components/common/index.ts:4]: What does this LOC do?

**Review by kntsoriano — COMMENTED** [review]
Looks good! I think we'd want the test where the react component and the view helper render the same thing.

**Comment by iHiD** [comment]
@kntsoriano Yes!! Thanks for remembering. I want that too.

So @ErikSchierboom , so the output of (haml):
```haml
=icon(:foobar)
```

Should be the same as the equiv React (once React has rendered):

```
<Icon icon="foobar">
```

**Comment by iHiD** [comment]
@ErikSchierboom Ignore the test for now, because it won't work for reasons I will explain to you, which https://github.com/exercism/v3-website/issues/251 will solve. So feel free to go ahead without the test, but create an issue for you to add it later pls?

File: app/javascript/components/common/index.ts (hunk 2 above)
Comment (ErikSchierboom) [reply app/javascript/components/common/index.ts:4]: It is a React convention for the `index.ts` to export all the components in the directory. You can then do the following to emulate some form of namespacing:

```jsx
import * as Common from '../components/common'

<Common.CopyToClipboardButton textToCopy={data.text_to_copy} />
```

**Comment by ErikSchierboom** [comment]
@iHiD I've created an issue: https://github.com/exercism/v3-website/issues/253

File: app/javascript/components/common/index.ts (hunk 2 above)
Comment (iHiD) [reply app/javascript/components/common/index.ts:4]: Ah, cool.

**Comment by iHiD** [comment]
This is red, but good to merge when green.

**Review by iHiD — APPROVED** [review]

**Pull request merged**